### PR TITLE
feature : Api Integration changes

### PR DESCRIPTION
**Pull request by sreeragmsurendran**
Api Integration changes

### CODE REVIEW

File: src/components/Listview/ListView.jsx [hunk 1]
```diff
@@ -1,18 +1,26 @@
...
 const ListView = ({ className }) => {
-    //mimic the API data with dummy data
-    let listItems = [
-        { item: "jkshjkhdhksd" }, { item: "jkshjkhdhksd" }, { item: "jkshjkhdhksd" }, { item: "jkshjkhdhksd" }, { item: "jkshjkhdhksd" }, { item: "jkshjkhdhksd" }, { item: "jkshjkhdhksd" }, { item: "jkshjkhdhksd" }, { item: "jkshjkhdhksd" }, { item: "jkshjkhdhksd" }, { item: "jkshjkhdhksd" }, { item: "jkshjkhdhksd" }]
 
-    // TODO: REMOVE listItems ONCE API DATA IS LOADED
+    const [data, setData] = useState([]);
+
+    useEffect(() => {
+        axios.get('/api/v1/todo/items')
```
Comment (RobinRz) [inline src/components/Listview/ListView.jsx:10]: set this url in urls and export from there. check urls.jsx for examples

Comment (RobinRz) [reply src/components/Listview/ListView.jsx:10]: urls is in src/utilities/
const todoReadManyUrl = userId =>
  `api/todos/v1/${userId}/todo`; will already be there. just customise it.

File: src/components/home/index.jsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import TitleBar from '../common/TitleBar';
 import SideBar from '../common/SideBar';
 import DefaultLayout from '../layouts/DefaultLayout';
-import ListView from '../ListView/ListView'
+import ListView from '../listview/ListView'
```
Comment (RobinRz) [inline src/components/home/index.jsx:4]: shouldn't the directory name be in camel case or ?

File: src/components/Listview/ListView.jsx (hunk 1 above)
Comment (sreeragmsurendran) [reply src/components/Listview/ListView.jsx:10]: Will consider that as future scope

File: src/components/home/index.jsx (hunk 2 above)
Comment (sreeragmsurendran) [reply src/components/home/index.jsx:4]: Restof the directory name is smallcase so that y changed

Comment (RobinRz) [reply src/components/home/index.jsx:4]: either camel case or seperated by underscores are what i followed for directories

**Pull request merged**